cms/templates/ux/reference/modal_access-component.html

```diff
@@ -0,0 +1,126 @@
+<div class="wrapper wrapper-modal-window wrapper-modal-window-edit-xblock" aria-describedby="modal-window-description" aria-labelledby="modal-window-title" aria-hidden="" role="dialog">
```

Comment:
```aria-describedby``` and ```aria-labelledby``` are non-conformant as they reference a non-existent ID (describedby) or a class name (labelledby).

Also, its critical that we move focus to this dialog when it appears on screen.  Adding a ```tabindex="-1"``` on this div and then calling ```focus()``` on the div should do the trick.  Ideally, you want to return focus to the element that triggered it (the eye/visibility icon) upon save or cancel.  You *should* also trap the keyboard in the dialog and add support for the ESC key (same as clicking the cancel button).